assets/app/scripts/controllers/pod.js

```diff
@@ -7,15 +7,16 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('PodController', function ($scope, $routeParams, DataService, project, $filter, ImageStreamResolver) {
+  .controller('PodController', function ($scope, $routeParams, $timeout, DataService, project, $filter, ImageStreamResolver) {
+
     $scope.pod = null;
     $scope.imageStreams = {};
     $scope.imagesByDockerReference = {};
     $scope.imageStreamImageRefByDockerReference = {}; // lets us determine if a particular container's docker image reference belongs to an imageStream
-    $scope.builds = {};     
+    $scope.builds = {};
```

Comment:
on the upside... looking forward to not having whitespace issues in out git diffs one day.